scripts/publish_zap_compatible.py

```diff
@@ -28,22 +28,6 @@ def scan_and_remove(obj, mappings):
 with open('build/communications-manager.json') as f:
     specification = json.load(f)
 
-# set the servers - this allows us to test unpublished environments
-specification["servers"] = [
```

Comment:
Does the removal of this actually render the openapi spec generated for Zap empty ? 

The communications-manager-zap.json ?

If so and this now still works, can we remove that code all together ?